SPECS/tpm2-pkcs11/tpm2-pkcs11.spec

```diff
@@ -0,0 +1,124 @@
+Summary:        OSS implementation of the TCG TPM2 Software Stack (TSS2)
+Name:           tpm2-pkcs11
+Version:        1.9.0
+Release:        1%{?dist}
+License:        BSD 2-Clause
+URL:            https://github.com/tpm2-software/tpm2-pkcs11
+Group:          System Environment/Security
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+
+Source0: https://github.com/tpm2-software/tpm2-pkcs11/releases/download/%{version}/%{name}-%{version}.tar.gz
+%define sha512 tpm2=006943b3853dc80e44d2322ea0278d6a9f2139c3b3e2a2c5f33436d479d698c5b9d685fb1166d22562bcf3d52edb1075efe7592c27a8c3a0cd05356cab3c9874
```

Comment:
this hash doesn't appear to match either our version (1.9.0) or Photon's version (1.8.0) so i have no idea what this hash line is supposed to represent.

Reply:
removed it